src/main/java/hudson/plugins/emailext/EmailRecipientUtils.java

```diff
@@ -132,10 +132,6 @@ public FormValidation validateFormRecipientList(String recipientList) {
     private static String fixupDelimiters(String input) {
         input = input.trim();
         input = input.replaceAll("\\s+", " ");
-        if(input.contains(" ") && !input.contains(",")) {
-            input = input.replace(" ", ",");
-        }
-
         input = input.replace(';', ',');
         return input;
     }
```

Comment:
How about this?

```java
private static String fixupDelimiters(String input) throws AddressException {
        input = input.trim();
        input = input.replaceAll("\\s+", " ");
        input = input.replaceAll(";", ",");
        if(input.contains(" ") && !input.contains(",") && !input.contains("<")) {
            // fix-up spaces to commas
            input = input.replaceAll(" ", ",");
        }
        return input;
    }
```

Reply:
You can't do that check comment with explanation:
https://github.com/jenkinsci/email-ext-plugin/pull/395#issuecomment-1269641355